modules/placement-driver/src/integrationTest/java/org/apache/ignite/internal/placementdriver/PlacementDriverManagerTest.java

```diff
@@ -321,6 +449,45 @@ private void checkLeaseCreated(TablePartitionId grpPartId) throws InterruptedExc
         }, 10_000));
     }
 
+    /**
+     * Checks that the lease for the group is accepted.
+     *
+     * @param grpPartId Replication group id.
+     * @throws InterruptedException If the waiting is interrupted.
+     */
+    private void checkAccepted(TablePartitionId grpPartId) throws InterruptedException {
+        assertTrue(waitForCondition(() -> {
```

Comment:
Hmm, just curious. Is it possible to rewrite it with `assertThat(leaseFut, willBe());` ?

Reply:
Here is a waiting of the lease creation. Of course, it is possible to create an infinity loop and complete a future after the loop found the lease, but that will be another code.